/**:
  ros__parameters:
    # -- Auto Park Node Configurations --
    th_arrived_distance_m: 1.0
    th_parking_space_distance_m: 10.0
    update_rate: 5.0
    vehicle_shape_margin_m: 0.2

    # -- Configurations common to the all planners --
    # base configs
    time_limit: 30000.0
    minimum_turning_radius: 5.0
    maximum_turning_radius: 9.0
    turning_radius_size: 3
    # search configs
    theta_size: 144
    angle_goal_range: 6.0
    curve_weight: 1.2
    reverse_weight: 2.0
    lateral_goal_range: 0.5
    longitudinal_goal_range: 2.0
    # costmap configs
    obstacle_threshold: 100

    # -- A* search Configurations --
    astar:
      only_behind_solutions: false
      use_back: true
      use_curve_weight: false
      use_complete_astar: true
      distance_heuristic_weight: 1.0
